SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Supported Versions
+
+Only the latest version is supported.
+
+## Reporting a Vulnerability
+
+If you believe you have found a security vulnerability in SciCat, please report it to us through coordinated disclosure.
+
+**Please do not report security vulnerabilities through public GitHub issues, discussions, or pull requests.**
+
+Instead, please email <ADDRESS>
```

Comment:
We still need to put an email address here.

Alternatively, or in addition, you can also enable private security reporting for the repo: https://docs.github.com/en/code-security/security-advisories/working-with-repository-security-advisories/configuring-private-vulnerability-reporting-for-a-repository

Reply:
@jl-wynen  Thanks for the info. I have enabled the security report feature for this repository. 
@nitrosx What email should we put in the template?